### ISSUE TITLE
Bootstrap version

### ISSUE THREAD

**Issue by CQer789** [anburocky3/bootstrap5-snippets#22]
Hi,
I downloaded this extension (very nice, by the way).
However, after starting up a page with the command bs5 <TAB> I see the stylesheet command and <script></script> tags are for Bootstrap 5.2.

Is there any possibility I can make it use Bootstrap 5.3, without entering this all manually, or is there an update available that can do this?

Thank you in advance.

**Comment by M3R14M** [comment]
I have updated my snippets to include version 5.3.2. Here's how to do it: 
Go to the folder `.vscode\extensions\anbuselvanrocky.bootstrap5-vscode-0.4.2\snippets`, open the file `snippets.code-snippets` in VS Code. Replace the `bs5-$` with this:

```json
"bs5-$": {
    "description": "Bootstrap master template",
    "scope": "",
    "prefix": "bs5-$",
    "body": "<!doctype html>\n<html lang=\"en\">\n\n<head>\n  <title>${1:Title}</title>\n  <!-- Required meta tags -->\n  <meta charset=\"utf-8\">\n  <meta name=\"viewport\" content=\"width=device-width, initial-scale=1, shrink-to-fit=no\">\n\n  <!-- Bootstrap CSS v5.3.2 -->\n  <link href=\"${2:https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css}\" rel=\"stylesheet\"\n    integrity=\"${3:sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN}\" crossorigin=\"anonymous\">\n\n</head>\n\n<body>\n  <header>\n    <!-- place navbar here -->\n  </header>\n  <main>\n\n  </main>\n  <footer>\n    <!-- place footer here -->\n  </footer>\n  <!-- Bootstrap JavaScript Libraries -->\n  <script src=\"${4:https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js}\"\n    integrity=\"${5:sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL}\" crossorigin=\"anonymous\">\n  </script>\n</body>\n\n</html>$0"
  }
  ```
  
  Ofcourse you can update this in the future with new CDN's. 
  
  No guarantee everything will work correctly.

**Comment by anburocky3** [comment]
@CQer789 @M3R14M Thanks for comments. Released in `v0.4.3`
